metadata.yaml

```diff
@@ -45,10 +45,15 @@ resources:
       Promql-transform binary for adding label selectors on amd64. Promql-transform is available
       as a stand-alone tool on https://github.com/canonical/promql-transform
     filename: promql-transform-amd64
+    # The version uploaded with the charm should be the one shipped as released in the
+    # https://github.com/canonical/promql-transform repository that matches the version
+    # of Prometheus shipped within the `prometheus-image` resource.
```

Comment:
This sounds a bit confusing

Reply:
Agreed. Tried to simplify the language a bit.